specs/sql-specs/community-service-api/function.user_details_test.sql

```diff
@@ -56,16 +60,13 @@ BEGIN;
         declare
             _result_row jsonb;
         begin
-            -- when scoped is owner of 
+            -- when anonymous
```

Comment:
anonymous should can access this endpoint, but not should see the address or some another details in future